### PR TITLE
node tuning operator - custom tuning

### PR DESCRIPTION
**Pull request by skordas**
Start using debug node instead jumping node.
New test case for custom tuning.
Moving some general methods to util file.

### CODE REVIEW

**Comment by mffiedler** [comment]
@hongkailiu ptal   @skordas sorry for missing this one in the queue

File: openshift_tooling/node_tuning_operator/node_tuning_operator_custom_tuning.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+execute_command("oc project openshift-cluster-node-tuning-operator")
+print_step("Getting current apiVersion")
+api_version = execute_command("oc get tuned default -o json | jq -r .apiVersion").rstrip()
+print("apiVersion: {}".format(api_version))
+
+# Getting all pods in project
+print_step("Getting information about pods")
+tuned_pods = execute_command("oc get pods --no-headers -o=custom-columns=NAME:.metadata.name | grep tuned").split("\n")
+del tuned_pods[-1]  # split method is giving extra empty field after last line from response
+passed("List of tuned nodes:\n" + str(tuned_pods))
```
Comment (hongkailiu) [inline openshift_tooling/node_tuning_operator/node_tuning_operator_custom_tuning.py]: nodes ---> pods
Line67 in **node_tuning_operator.py**: similar.

File: openshift_tooling/node_tuning_operator/node_tuning_operator.py [hunk 2]
```diff
@@ -26,51 +16,6 @@
 reset = "\33[0m"
```
[the remark this answers is not in the document]
Comment (skordas) [reply openshift_tooling/node_tuning_operator/node_tuning_operator.py]: Colors definitions are in util file

**Comment by hongkailiu** [comment]
This has not been addressed yet.
> This applies to other before/after comparison.

**Pull request merged**